src/System.Reflection.MetadataLoadContext/src/System.Reflection.MetadataLoadContext.csproj

```diff
@@ -0,0 +1,174 @@
+﻿<!-- System.Reflection.MetadataLoadContext.dll has two build configurations: NetCore and NetStandard. The functional difference is that the NetCore
+  version supports all of the new NetCore Reflection apis (IsSZArray, etc.) Apps built against NetCore should use the NetCore build
+  of MetadataLoadContext.dll if possible. If they use the NetStandard version, they must refrain from calling NetCore-specific apis on Reflection
+  objects dispensed by MetadataLoadContexts as these apis will not work.
+    -->
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <ProjectGuid>{DBB18588-DD89-4F41-9852-484FD4726F0F}</ProjectGuid>
+    <AssemblyName>System.Reflection.MetadataLoadContext</AssemblyName>
+    <RootNamespace>System.Reflection</RootNamespace>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Description>.NET Core Type Loader</Description>
+    <Configurations>netcoreapp-Debug;netcoreapp-Release;netstandard-Debug;netstandard-Release</Configurations>
+  </PropertyGroup>
+
+<!-- The ItemGroup represents types originally obtained from CoreRt. Keep the FromCoreRt path as we may want to move these
```

Comment:
This is pretty weak reason. Even if everything is in the same directory, you can trivially get up-to-date list of files that are candidates for sharing with CoreRT by comparing file list.

(If you really want to keep the CoreRT folder, it should be either all lower case `corert` or cased as `CoreRT`. We do not use `CoreRt` casing anywhere.)